Helper/OmiseHelper.php

```diff
@@ -276,6 +276,36 @@ public function omiseAmountFormat($currency, $amount)
         return $amount;
     }
 
+    /**
+     * @param  string  $currency
+     * @param  integer $amount
+     *
+     * @return string
+     */
+    public function currencyAmountFormat($currency, $amount)
+    {
+        $supportedCurrencies = [
```

Comment:
Is this an array of `supportedCurrencies` or an array of `nonZeroDecimalCurrencies`? https://www.omise.co/currency-and-amount#units